packages/core/src/divide/halfUp.ts

```diff
@@ -1,5 +1,5 @@
 import type { DivideOperation } from '..';
-import { greaterThan, isHalf, absolute } from '../utils';
+import { absolute, greaterThan, isHalf } from '../utils';
```

Comment:
Is this edit necessary?

Reply:
Every other import in every other file I've touched is sorted in alphabetical order except this. XD

Don't feel strongly about it though. Should probably be added as a lint if people care. My understanding is that the order of imports from different files matters in JS but not multiple imports from the same file - if this is not correct, then the change should be reverted.

Reply:
Understood, I would agree if alphabetical order is important we should add an eslint rule for it.